routes/index.js

```diff
@@ -3,8 +3,8 @@ var app=express()
 var mongoose = require('mongoose');
 var router = express.Router();  //creatig insatnce of express function
 var crypto =require('crypto');
-
-router.post('/user/register', function(req, res) {  
+var moment=require("moment");
```

Comment:
always install modules using npm install moment --save
so that package.json is also updated.  fix this so that package.json is also update